properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -431,10 +431,9 @@ making sure that you maintain a proper JSON format.
 					{
 						"propertyName": "Interval",
 						"propertyKeyword": "interval",
-						"propertyType": "numeric",
-						"valueType": "number",
-						"allowNegative": false,
-						"step": 1,
+						"propertyType": "details",
```

Comment:
That's probably bad, we may still use it for polyglot-oracle conversion so we may still need numeric here